### PR TITLE
resolved bug in the POST articles/gifs

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

This PR resolves the bug in the POST articles/gifs endpoints
#### How should this be manually tested?

1. Make a POST request with sample data below to this endpoint `http://localhost:3000/api/v1/articles/gifs`
```
{
    "title" : "carl is",
    "article" : "https://ibb.co/cFytKDw"
}

```
3. This successfully creates an article/gif
#### What are the relevant issues?
we had to resolve the bug in the POST articles/gifs endpoint

### CODE REVIEW

**Comment by beblicarl** [comment]
@jocrah , kindly review this for me

File: src/routes/articles/create-article.e2e.test.js [hunk 1]
```diff
@@ -14,26 +14,6 @@ describe('POST /articles', () => {
 		)
```
Comment (jocrah) [inline src/routes/articles/create-article.e2e.test.js:14]: Kindly update any other part of this test that is sending `userId` as part of the request body

Comment (beblicarl) [reply src/routes/articles/create-article.e2e.test.js:14]: sure

Comment (beblicarl) [reply src/routes/articles/create-article.e2e.test.js:14]: I have implemented the necessary changes

**Review by jocrah — APPROVED** [review]

**Comment by jocrah** [comment]
@beblicarl  in light of this issue, we might need some consistency in our requests then

if we can get the userId from the req.user, we shouldn't have userId in most of our other schemas

**Pull request merged**